pagefetcher.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python2
+
+import urllib
+import os
+from __builtin__ import str
+
+# set of dictionaries to store url's and names
+CISCO = {"name1": "url1", "name2": "url2"}
```

Comment:
we need the name of the file to be saved. see my earlier comment
> Here is some of the options I thought about:
1. use dictionary

>cisco = {"name": "url"}
template_to_url = {"cisco": cisco}

>one advantage of using this method is that it's easy to use and clean.
>2. use list (version 1)

>cisco = ["name", "url"]
>template_to_url = {"cisco": cisco}

>this option requires a bit of extra looping to go through.
>3. use a list (version 2)

>cisco = ["name:url"]
template_to_url = {"cisco": cisco}